packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -348,9 +361,10 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     [attachmentOptions?.downloadOptions?.actionsForAttachment]
   );
 
+  const threadId = useSelector(getThreadId);
```

Comment:
Should we group all values created with `useSelector` at the top of the components?

Reply:
Is a preference thing, I like seeing variables defined right before they're used but happy to move if you prefer to group them at the top

Reply:
I'm okay with both approaches :)